Makefile

```diff
@@ -210,6 +218,31 @@ endif
 image/build:
 	${DOCKER} build . -f ${DOCKERFILE} -t ${IMG}:$(IMG_VERSION)
 
+# build docker image using buildx
+# PLATFORMS defines the target platforms for the model registry image be built to provide support to multiple
+# architectures. (i.e. make docker-buildx). To use this option you need to:
+# - be able to use docker buildx. More info: https://docs.docker.com/build/buildx/
+# - have enabled BuildKit. More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# - be able to push the image to your registry (i.e. if you do not set a valid value via IMG=<myregistry/image:<tag>> then the export will fail)
```

Comment:
can't we make a separate rule for pushing? we're not only building for that purpose AFAIK, but also to be able to have those images locally.

Reply:
Docker doesn't support that, it doesn't keep manifests locally and have to be pushed to a registry.

Reply:
you can use `--load` on the build.

Reply:
There are open issues with `--load` and it didn't work for me. Multi-platform builds are useful for publishing to remote repos, not as much for local use. 
E.g. we'll do it in  CI on PR merge and not for pulls.